README.md

```diff
@@ -1,13 +1,18 @@
 # Project Movies
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The goal of this assignment was to fetch movie data and present it as a multi page application.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We started by creating components for different pages. Because this was a pair programming assignment we planned early in the week on how to use our time most efficiently.
```

Comment:
Sounds like you had a good structure for your time management and also that you took it step by step, nice job